azure-pipelines.yml

```diff
@@ -215,32 +215,28 @@ jobs:
       # * It excludes targets that depend on bitstream_splice rules, since the
       #   environment does not have access to Vivado.
       export GCP_BAZEL_CACHE_KEY=$(bazelCacheGcpKeyPath)
-      TARGET_PATTERN_FILE=$(mktemp)
```

Comment:
Would it make sense to instead chnage this to
`TARGET_PATTERN_FILE=target_pattern.txt`
so that the diff is smaller and this can be changed more easily in we need to?

Reply:
I've set `TARGET_PATTERN_FILE` to make the diff smaller, but I had trouble with using Azure's variables in both bash scripts and in the pipeline definition previously.

I'm certain it can be done, I just kept messing it up and the feedback cycle is 3 hours so I gave up for now :)